evalml/automl/automl_search.py

```diff
@@ -693,8 +695,8 @@ def _post_evaluation_callback(self, pipeline, evaluation_results, job_log):
         self._results['pipeline_results'][pipeline_id] = {
             "id": pipeline_id,
             "pipeline_name": pipeline.name,
-            "pipeline_class": type(pipeline),
-            "pipeline_summary": pipeline.summary,
+            'pipeline': pipeline,
```

Comment:
Renaming to `pipeline` to indicate that it's an instance, not the class, that we're dealing with.